src/server/services/procedures/mauna-loa-co2-data/mauna-loa-co2-data.js

```diff
@@ -7,24 +7,20 @@
  * @category Science
  * @category Climate
  */
+
+const fs = require('fs');
+const path = require('path');
+
 const data = (function() {
-    const fs = require('fs');
-    const path = require('path');
     const filename = path.join(__dirname,'co2_mm_mlo.txt');
     const lines = fs.readFileSync(filename, 'utf8').split('\n');
-
-    while (lines[0].startsWith('#')) {
-        lines.shift();
-    }
-    return lines.map(line => {
-        const [,, date, avg, interpolated, trend] = line.split(/\s+/)
-            .map(txt => parseFloat(txt));
-        return {date, avg, interpolated, trend};
+    return lines.filter(x => !x.startsWith('#')).map(line => {
+        const [,, date, interpolated, trend] = line.split(/\s+/).map(parseFloat);
+        return {date, interpolated, trend};
     });
 })();
 
 const MaunaLoaCO2Data = {};
-MaunaLoaCO2Data.serviceName = 'MaunaLoaCO2Data';
```

Comment:
Why did you remove this line? Won't this change the name to "MaunaLoaCo2Data" (note the lowercase "o")?

Reply:
Oh, yeah. Ok, I just fixed that. I must've been thinking NetsBlox got the name from the export object name when it does the jsdoc parsing, so it looked redundant, but it actually gets it from the folder/file.